src/modpacks/models.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+from django.db import models
+
+
+class Mod(models.Model):
+    """
+    A mod that can be combined to make a modpack
+    """
+    mod_name = models.CharField(verbose_name='Mod name',
+                                max_length=200)
+    version = models.CharField(verbose_name='Mod version',
+                               blank=True,
+                               max_length=10)
+    author = models.CharField(verbose_name='Mod author',
+                              blank=True,
+                              max_length=200)
+    url = models.URLField(verbose_name='Link to mod website',
+                          blank=True)
+
+    def __str__(self):
+        return self.mod_name
+
+
+class Modpack(models.Model):
```

Comment:
In other frameworks, all of the models live in their own library (.dll or .so). I realize that doesn't fit well with the django pattern; it'd take some finessing to make it work.

Actually, the new hotness is to loosely couple models. You'd have two model libraries, data models and view models. They're coupled via a mapper, which knows how to translate data from a database (data models) into data used by the front-end (view models). This is the MVVM pattern. It's useful when you don't need to store everything about the front-end in the database, IE: you store a lookup key/query string and hit a 3rd party API for the rest of the data. This pattern also allows you to change the schema in either the front-end or the back end without affecting the other's model; only need to change the mapper.

We can talk via hangouts if you're interested in this pattern... but mvvm doesn't look like something explicitly supported by django.